### PR TITLE
Migrate acceptance tests to prow

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Added makefile for acceptance-tests
- Added go vet and goimports checks to the `build.sh` script

**Related issue(s)**

### CODE REVIEW

File: tests/acceptance/README.md [hunk 1]
```diff
@@ -6,12 +6,16 @@ This project contains the acceptance tests that you can run as part of Kyma test
 The tests are written in Go. Each component or group of scenarios has a separate folder, like `dex` or `servicecatalog`.
 
 ## Usage
+
 This section provides information on building and versioning of the Docker image, as well as configuring Kyma.
+Use `make build build-image` to test your changes and build image.
```
Comment (klaudiagrz) [inline tests/acceptance/README.md]:
```suggestion
To test your changes and build an image, use the `make build build-image` command. You can also add other tests. To add a new test:
```

File: tests/acceptance/README.md [hunk 2]
```diff
@@ -6,12 +6,16 @@ This project contains the acceptance tests that you can run as part of Kyma test
...
 
 ### Adding new tests
```
Comment (klaudiagrz) [inline tests/acceptance/README.md]: Remove this heading

File: tests/acceptance/README.md [hunk 3]
```diff
@@ -6,12 +6,16 @@ This project contains the acceptance tests that you can run as part of Kyma test
...
+
 1. Add a new package
 2. Modify the Dockerfile to compile the test binary to pkg.test
 3. Add execution of the test to the `entrypoint.sh` script
+4. Add deletion of the binary to Makefile's cleanup step.
```
Comment (klaudiagrz) [inline tests/acceptance/README.md]: Let's discuss

File: tests/acceptance/README.md [hunk 4]
```diff
@@ -6,13 +6,18 @@ This project contains the acceptance tests that you can run as part of Kyma test
 The tests are written in Go. Each component or group of scenarios has a separate folder, like `dex` or `servicecatalog`.
 
 ## Usage
-This section provides information on building and versioning of the Docker image, as well as configuring Kyma.
 
-### Adding new tests
-1. Add a new package
-2. Modify the Dockerfile to compile the test binary to pkg.test
-3. Add execution of the test to the `entrypoint.sh` script
+To test your changes and build an image, use the `make build build-image` command. You can also add new tests.
```
Comment (aszecowka) [inline tests/acceptance/README.md]: > You can also add new tests. 

?

File: tests/acceptance/README.md [hunk 5]
```diff
@@ -6,13 +6,18 @@ This project contains the acceptance tests that you can run as part of Kyma test
...
 
-### Configuring Kyma
+### Add new test
+
+To add a new test:
+
+1. Add a new package.
+2. Modify the Dockerfile to compile the test binary to pkg.test.
```
Comment (aszecowka) [inline tests/acceptance/README.md]: build.sh also will be modified

File: tests/acceptance/build.sh [hunk 6]
```diff
@@ -26,6 +26,41 @@ if [ $(echo ${#fmtResult}) != 0 ]
...
+if [ $(echo ${#goImportsResult}) != 0 ]; then
+	echo -e "${RED}✗ goimports and fmt${NC}\n$goImportsResult${NC}"
+	exit 1
+else echo -e "${GREEN}√ goimports and fmt${NC}"
+fi
+
+##
+# GO VET
+##
+packagesToVet=("./dex/..." "./servicecatalog/..." "./remote-environment/...")
```
Comment (aszecowka) [inline tests/acceptance/build.sh]: if we add more tests, we also need to change that. Please check if it is another option to do that.

**Pull request merged**